pkg/cmd/server/origin/aggregator.go

```diff
@@ -42,6 +42,13 @@ import (
 	informers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
 )
 
+func init() {
+	// install our types into the scheme so that "normal" RESTOptionsGetters can work for us.
+	// done in init() so we don't mutate the scheme after it is being used by clients in other goroutines.
+	// TODO: move to a scheme-per-process like upstream and register into that instead.
+	install.Install(kapi.GroupFactoryRegistry, kapi.Registry, kapi.Scheme)
```

Comment:
We really don't want to do this.  If your integration tests requires it, do it there.  You shouldn't be relying on a side effect of loading the package and if anyone gets used to it, we'll be chasing problems around as people who transitively depend on this package but never starts a server will accidentally work.